JS/see_article.js

```diff
@@ -172,8 +192,25 @@ function create_see_article(article) {
         }
         if (article_already_in_cart == false) {
```

Comment:
do I have to says more ?

Reply:
There is no tab anymore

Reply:
`article_already_in_cart == false` => `!article_already_in_cart`

Reply:
changed